### PR TITLE
data: NishantCoder108

### PR DESCRIPTION
**Pull request by NishantCoder108**

### CODE REVIEW

**Review by NishantCoder108 — COMMENTED** [review]
Successfully contribute.

**Comment by NishantCoder108** [comment]
linkfree link is not created.

File: data/NishantCoder108.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		},
+     {
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/nishantcoder/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Follow me on YouTube",
+			"url": "https://www.youtube.com/channel/UCrx_pTOxfgCLGAS7aYzG9iQ",
+			"icon": "FaYouTube"
```
Comment (khushalarora) [inline data/NishantCoder108.json:23]: Please replace this with **' FaYoutube '** :+1:

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Comment by SaraJaoude** [comment]
> linkfree link is not created.

As mentioned in our Docs once you create your Pull Request you need to wait for a maintainer to approve this: https://linkfree.eddiehub.io/docs/quickstart

Screenshot 2023-04-04 at 09 01 31

File: data/NishantCoder108.json (hunk 1 above)
Comment (SaraJaoude) [reply data/NishantCoder108.json:23]: This is not a blocker so I will be merging this Pull Request.

Any changes or amendments to the Profile can be raised in future in a new Pull Request and you may wish to refer to the Editing Guide: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**